config/observability/README.md

```diff
@@ -0,0 +1,48 @@
+# Observability stack guide
+
```

Comment:
I see what you're getting at.
I'm thinking this is a developer (us) focused readme for when we want to work on the example dashboards (or alerts eventually), rather than a user focused walkthrough.
There's a separate issue for documenting how a user would consume the dashboards. That may or may not be written as a walkthrough (TBD).
With this context, I'm thinking it's OK to not be specific here as it may get out of date without us realising, which the instructiions as they stand now should hold up longer as they don't go into specific detail on how to get to the point where you can modify dashboards.